data-raw/watershed/american/american_river.rmd

```diff
@@ -88,50 +104,59 @@ american_river_instream %>%
   geom_line() +
   labs(x = 'Flow (cfs)', y = 'WUA (sqft/1000ft)') + 
   theme_minimal() + 
-  scale_color_manual(values = c('#d95f02','#7570b3'))
+  scale_color_manual(values = c('#d95f02','#7570b3')) +
+  theme(legend.justification = c(1,0), legend.position = c(.9,.7))
 
 ```
-
 ```{r, include=FALSE}
-source('data-raw/floodplain/floodplain_utils.R')
+source('data-raw/watershed/floodplain/floodplain_utils.R')
 ```
 
-## Floodplain Modeling Details
-
+## Floodplain Rearing Habitat
 **Data Source:**
 [Central Valley Floodplain Evaluation and Delineation (CVFED) HEC-RAS hydraulic model](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/CombinedTM_IQAR_Final-FULL-REPORT_20140206.pdf)
 
 `r print_model_details('American River', 'fr')`
 
 ## Floodplain Data
-
 ```{r, echo=FALSE}
-american_river_floodplain <- read_excel('data-raw/floodplain/CVPIA_FloodplainAreas.xlsx',
-                                        sheet = 'AmericanRiver') %>%
+american_river <- read_excel('data-raw/watershed/floodplain/CVPIA_FloodplainAreas.xlsx',
+                                        sheet = 'AmericanRiver') 
+
+threshold_flow <- american_river %>% 
+  filter(modeled_floodplain_area_acres == 0) %>% 
+  summarise(max_flow = max(flow_cfs)) %>% 
+  pull(max_flow)  
+  
+american_river_floodplain <- american_river %>% 
   mutate(watershed = 'American River',
          SR_floodplain_acres = modeled_floodplain_area_acres,
          ST_floodplain_acres = modeled_floodplain_area_acres) %>%
   select(flow_cfs,
          FR_floodplain_acres = modeled_floodplain_area_acres,
          SR_floodplain_acres, ST_floodplain_acres,
-         watershed)
+         watershed) %>% 
+  filter(flow_cfs >= threshold_flow)
 
 knitr::kable(align = 'c', head(american_river_floodplain, 5))
 # devtools::use_data(american_river_floodplain, overwrite = TRUE)
```

Comment:
replace `devtools` package with `usethis`, add that check to your checklist

Reply:
fixed usethis